src/pages/iou/request/MoneyTemporaryForRefactorRequestParticipantsSelector.js

```diff
@@ -380,4 +383,14 @@ export default withOnyx({
     betas: {
         key: ONYXKEYS.BETAS,
     },
-})(MoneyTemporaryForRefactorRequestParticipantsSelector);
+})(
+    memo(
+        MoneyTemporaryForRefactorRequestParticipantsSelector,
+        (prevProps, nextProps) =>
+            _.isEqual(prevProps.participants, nextProps.participants) &&
+            prevProps.didScreenTransitionEnd === nextProps.didScreenTransitionEnd &&
```

Comment:
@TMisiukiewicz Is there a particular reason we're checking `didScreenTransitionEnd` here, especially as it never fires on Android?

Reply:
in this page, `didScreenTransitionEnd` is used to prevent generating options unless the transition ended, to avoid doing heavy calculations while animation is happening. Without checking it, we'd infinitely get stuck on the skeleton